src/js/_enqueues/vendor/codemirror/csslint.js

```diff
@@ -252,11 +252,11 @@ var Parser = require("./Parser");
  * Represents a selector combinator (whitespace, +, >).
```

Comment:
This file should be covered by the `.eslintignore` exclusion for this ` src/js/_enqueues/vendor` path, as such no changes to this file should be in this PR

https://github.com/WordPress/wordpress-develop/pull/418/files#diff-1b0fd5322de42e11c8786543f74c7c41R11